pkg/okteto/client.go

```diff
@@ -233,36 +230,3 @@ func (c *OktetoClient) Pipeline() types.PipelineInterface {
 func (c *OktetoClient) User() types.UserInterface {
 	return c.user
 }
-
-type traceTransport struct {
```

Comment:
iirc correctly this was some utility included by @maroshii to debug issues. Fran, am I right? is this needed/helpful?

Reply:
I'm in favor of keeping it if it's useful. We can add linting annotations so that we don't get the linting
https://golangci-lint.run/usage/false-positives/#nolint-directive

Reply:
I've never used it, so not sure how useful it is, I guess we can always document it in Notion as a helper and deleted. As I've never used it, my vote would be to delete it, but I'm ok also with adding the `nolint` directive just in case

Reply:
That'd be good. @maroshii could you explain how was this helpful when debugging?